opencti-platform/opencti-graphql/tests/utils/testQueryHelper.ts

```diff
@@ -70,6 +70,22 @@ export const queryAsUserIsExpectedForbidden = async (client: AxiosInstance, requ
   expect(queryResult.errors[0].extensions.code, `FORBIDDEN_ACCESS is expected but got ${queryResult.errors[0].name}`).toBe(FORBIDDEN_ACCESS);
 };
 
+/**
+ * Execute the query as some User (see testQuery.ts), and verify that error is thrown.
+ * @param client
+ * @param request
+ * @param errorMessage
+ * @param errorName
+ */
+export const queryAsUserIsExpectedError = async (client: AxiosInstance, request: any, errorMessage?: string, errorName?: string) => {
+  const queryResult = await executeInternalQuery(client, print(request.query), request.variables);
+  logApp.info('queryAsUserIsExpectedError=> queryResult:', queryResult);
+  expect(queryResult.errors, 'error is expected.').toBeDefined();
+  expect(queryResult.errors?.length, `1 error is expected, but got ${queryResult.errors?.length} errors`).toBe(1);
+  expect(queryResult.errors[0].message, `error message: ${errorMessage} is expected, but got ${queryResult.errors[0].message}`).toBe(errorMessage);
+  expect(queryResult.errors[0].extensions.code, `error is expected but got ${queryResult.errors[0].name}`).toBe(errorName);
```

Comment:
What is the behavior if we are not giving params `errorMessage` or `errorName` as they are optional?